examples/values-sigstore-openshift.yaml

```diff
@@ -55,6 +55,9 @@ configs:
           szmI+ON/YItgbv0e5Wu7t9700ujBNfgMhzyO
           -----END CERTIFICATE-----
 
+  cosign:
+    BASE_HOSTNAME: apps.rosa.p9esn-qgkm3-wkk.o7au.p3.openshiftapps.com
```

Comment:
i know it's annoying but to keep with the rest of this file can you change line 59 to: 
`BASE_DOMAIN: apps.<OPENSHIFT_BASE_DOMAIN>`
(the other file should use this too, but the values-ez assumes you're running in a ROSA cluster-bot cluster - bc it's convenient for me :) since I'm usually running one)